Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Button.css';
+
+const ActionButton = props => {
+    return (
+        <button className={`bigger-button`}>
```

Comment:
you can simply say className="bigger-button" here since it's just a string, no curlys needed